docs/pre-release-disclosure.md

```diff
@@ -20,8 +19,8 @@ this information prior to the public announcement date of the vulnerability.
 
 ## Disclosure Agreement
 
-Members on the cpan-security@perl.org mailing list need to agree to the
-following:
+Members of need to agree to the following before getting access to information
```

Comment:
maybe remove the "of" - Members need ...

Reply:
Thx!